plugins/render/render.go

```diff
@@ -0,0 +1,34 @@
+package render
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"os"
+)
+
+func ConfigPath() string {
+	configDir, err := os.UserConfigDir()
+	if err != nil {
+		return "~/.render/config.yaml"
+	}
+	return configDir + "/render/config.yaml"
+}
```

Comment:
Let's remove this function - multiple tools depend on the user configuration directory, which we are planning to add support for at a later time.

Ensuring that each tool doesn't handle this logic in its own way is going to make it easier to migrate. For the time being, I would just hardcode "~/.render/config.yaml" in `provision.AtFixedPath`.

Reply:
resolved